commcare_connect/opportunity/api/serializers.py

```diff
@@ -133,6 +138,8 @@ class Meta:
             "payment_units",
             "is_user_suspended",
             "catchment_areas",
+            "start_time_threshold",
```

Comment:
Is there a specific reason to change the names here? It obviously doesn't really matter but it does add a tiny bit of inconsistency between what the mobile and web team will call these which can be annoying.